pkg/proxy/iptables/proxier.go

```diff
@@ -967,6 +967,11 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 
+	// These two variables are used to publish the sync_proxy_rules_no_endpoints_total
+	// metric.
+	serviceNoEndpointsTotalInternal := 0
+	serviceNoEndpointsTotalExternal := 0
```

Comment:
Might be good to rename this to `serviceNoLocalEndpointsTotalExternal` and `serviceNoLocalEndpointsTotalInternal`

Reply:
Done